src/schema/v2/infiniteDiscovery/discoverArtworks.ts

```diff
@@ -48,19 +51,22 @@ export const DiscoverArtworks: GraphQLFieldConfig<void, ResolverContext> = {
         },
       }),
     },
-    useRelatedArtworks: { type: GraphQLBoolean, defaultValue: false },
+    useOpenSearch: { type: GraphQLBoolean, defaultValue: false },
+    excludeArtworkIds: {
+      type: new GraphQLList(GraphQLString),
+      description:
+        "(Only for when useOpenSearch is true) Exclude these artworks from the response, currently works only in combination with tasteProfileVector",
```

Comment:
**question:** Is this comment still relevant, now that we don't have the tasteProfileVector?

Reply:
thanks, not relevant anymore